pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1449,29 +1522,44 @@ func handleAppNetworkModify(ctxArg interface{}, key string,
 	// configure the ipsets for all the domU's interfaces/bridges.
 	ipsets := compileAppInstanceIpsets(ctx, config.UnderlayNetworkList)
 
+	// Look for Network changes in underlay
+	intfList, err := doAppNetworkModifyUNetAppNums(ctx, config,
+		oldConfig, status, ipsets)
```

Comment:
See elsewhere, but why can't this call handle all the changes due to the network instance UUID change?